### DISCUSSION TITLE
Runtime issues appeared on Windows runners

### DISCUSSION THREAD

**Discussion by krasznaa** [community/community#128019] [Actions]
### Select Topic Area

Bug

### Body

Hi,

We started seeing a very strange set of runtime issues with the unit tests of one of our projects, on Windows. See for instance this run:

https://github.com/krasznaa/vecmem/actions/runs/9449398840/job/26025548608

What I did there was to re-trigger the pipeline on an existing hash using a new branch, which ran about one month ago as well. See:

https://github.com/krasznaa/vecmem/actions

When I compile the project locally with the exact same version of MSVC that the CI is reporting, I cannot reproduce this issue. 😕 So it seems that something is wrong / different on the Windows nodes than what I would use locally.

Is anybody else seeing issues similar to this on Windows? Any good tips on how to debug this further? It would seem silly to introduce a script that would run a debugger as part of the Windows build on the GitHub runner. 🤔

Cheers,
        Attila

P.S. Pinging some affected PRs: https://github.com/acts-project/vecmem/pull/277, https://github.com/acts-project/vecmem/pull/276.

**Comment by Rishi-Sudhakar** [comment]
Review the compiler flags and build configurations, They're annoying to deal with. But since you said the issue can’t be reproduced locally, the issue might be probably in config. Also check if all dependencies are configured in the github env.

**Comment by krasznaa** [comment]
Hi @Rishi-Sudhakar,

In the meanwhile I'm quite certain that I ran into: https://github.com/actions/runner-images/issues/10004 Which is also discussed for instance here: https://github.com/google/googletest/issues/4556

So for now I'm waiting to see what would happen in those issues. 🤔

Attila
